try-catch/manifest.json

```diff
@@ -1,44 +1,25 @@
 {
     "title": "Try-Catch: Handle exceptions with the Scope action",
     "description": "This workflow template uses the 'Scope' action and Try-Catch pattern for exception handling.",
-    "tags": [
-        "Exception Handling",
-        "try catch"
-    ],
+    "artifacts": [],
     "skus": [
         "consumption",
         "standard"
     ],
-    "kinds": [
-        "stateful",
-        "stateless"
-    ],
-    "detailsDescription": "This workflow runs the specified actions that you add to represent business logic. If any errors occur, the workflow uses the 'Scope' action to catch and process those errors, which then terminates the workflow with a detailed error message. Learn more about [exception handling in Azure Logic Apps](https://go.microsoft.com/fwlink/?linkid=2226310).",
+    "workflows": {
+        "try-catch": {
+            "name": "try_Catch"
+        }
+    },
+    "featuredConnectors": [],
```

Comment:
You are also missing the Scope connector here, because that is what we were showing in the card. Just pick up the Scope connector ID from designer code and use here. you might need to support another connectorTYpe calle builtIN here